content/en/docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -66,9 +66,16 @@ kubectl get nodes
 Next, tell Kubernetes to drain the node:
 
 ```shell
-kubectl drain <node name>
+kubectl drain --ignore-daemonsets <node name>
 ```
 
+If there are DaemonSet managed pods, drain will usually not succeed unless you specify
```

Comment:
IMO, this sounds better and idiomatic without taking away from the meaning. WDYT?
```suggestion
If there are pods managed by a DaemonSet, you will need to specify
```

Reply:
yes, your wording is easier to read. Thank you.